### DISCUSSION TITLE
Not able to start a development server locally

### DISCUSSION THREAD

**Discussion by pratikkarad** [carbon-design-system/carbon#11042] [Help] [answered]
Hi team, I followed the 5 steps mentioned in the [contributing guidelines](https://github.com/carbon-design-system/carbon/blob/main/.github/CONTRIBUTING.md#start-contributing) and am still not able to start a development server locally. I have also attached snapshots of my console logs. On running yarn storybook, it is infinitely stuck on the console. I guess build itself is not happening correctly but I could be wrong. Any suggestions?

1. yarn install -
![yarn install](https://user-images.githubusercontent.com/32093370/159627226-bb0d8aea-db02-4d94-83fa-50b000121cb2.png)
2. yarn build -
![yarn build](https://user-images.githubusercontent.com/32093370/159627219-cf91df6b-b7f1-44c1-bbe2-3e54b991f2f3.png)
3. change directory to packages/react & yarn storybook -
![yarn storybook](https://user-images.githubusercontent.com/32093370/159627228-91447549-e9db-488b-a008-7bd336df5c61.png)

**Comment by tay1orjones** [comment, marked as answer]
@pratikkarad Sorry this isn't linked in the `contributing.md`, but there are some specific steps you must take to get things up and running in Windows:

https://github.com/carbon-design-system/carbon/blob/main/docs/guides/setup/windows.md

**Reply by tay1orjones** [reply]
https://github.com/carbon-design-system/carbon/pull/11044
